Molio/Source/Presentation/SwipeMusic/Model/SwipeMusicTrackModel.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+
+/// Swipe 할 수 있는 화면에 표시되는 음악 정보에 대한 UI View Model입니다.
+struct SwipeMusicTrackModel {
+    /// 노래 제목
+    let title: String
+    
+    /// 아티스트 이름
+    let artistName: String
+    
+    /// 장르 목록
+    let gerneNames: [String]
+    
+    /// 국제 표준 녹음 자료 코드
+    let isrc: String
+    
+    /// 음악 미리듣기 URL
+    let previewAsset: URL
+    
+    /// 앨범 아트워크 이미지
+    let artworkImageData: Data?
+    
+    /// 앨범에 이미지에 따른 평균 배경색
+    let artworkBackgroundColor: RGBAColor?
+    
+    /// 앨범에 이미지에 따른 primary 색상
+    let primaryTextColor: RGBAColor?
+    
+    init(randomMusic: RandomMusic, data: Data?) {
```

Comment:
P3;
외부에서 보기에 어떤 data인지 헷갈릴 때도 있을 것 같은데 (없을 수도 있습니다 ㅎ)
혹시 imageData로 매개변수 이름을 바꾸면 어떨까요??

물론 지금도 좋습니다 ㅎㅎ

```suggestion
    init(randomMusic: RandomMusic, imageData: Data?) {
```

Reply:
외부에서 보기에 헷갈릴 것 같습니다!! 수정완료했습니다 !!